.github/workflows/go.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         echo "::set-output name=go-mod::$(go env GOMODCACHE)"
 
     - name: Install encore-go
-      run: curl --fail -L -o encore-go.tar.gz https://github.com/encoredev/go/releases/download/encore-go1.18.4/linux_x86-64.tar.gz && tar -C . -xzf ./encore-go.tar.gz
+      run: curl --fail -L -o encore-go.tar.gz https://github.com/encoredev/go/releases/download/encore-go1.19.2/linux_x86-64.tar.gz && tar -C . -xzf ./encore-go.tar.gz
```

Comment:
Encore now uses go 1.19 features itself (`atomic.Uint64` / `atomic.Bool`)

Even though our `go.mod` file said 1.19, and we've told github at the tp of this file to use Go 1.19 - we ran tests using our older 1.18 fork